app/assets/javascripts/application.js

```diff
@@ -13,4 +13,12 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
-//= require_tree .
+//= require_tree
+= require bootstrap-dropdown
+= require bootstrap
+
+// jQuery(function($) {
+// $("tr[data-link]").click(function() {
+// window.location = this.dataset.link
+// });
+// })
```

Comment:
I think we needed it and it was working locally, but got messed up when we pushed to heroku.